src/components/SelfUser.vue

```diff
@@ -31,13 +34,16 @@ export default {
                 str = '+' + str;
             }
 
-            return str;
+            return 'User modes: ' + str;
```

Comment:
The 'User modes: ' text isn't needed here as it's obvious what it is already while keeping it less cluttered

Reply:
im not convinced this is clear how it originally was for me it was just a line with +cx but no explanation of that +cx is

Reply:
![image](https://user-images.githubusercontent.com/558946/41413193-3a869b74-6fda-11e8-8bf5-d5e2d6099de4.png)